ml-agents-envs/mlagents_envs/base_env.py

```diff
@@ -142,8 +149,10 @@ def empty(spec: "BehaviorSpec") -> "DecisionSteps":
         return DecisionSteps(
             obs=obs,
             reward=np.zeros(0, dtype=np.float32),
+            team_reward=np.zeros(0, dtype=np.float32),
             agent_id=np.zeros(0, dtype=np.int32),
             action_mask=None,
+            team_manager_id=np.zeros(0, dtype=np.int32),
```

Comment:
Isn't the default for team_manager_id -1 according to the C# code ?

Reply:
fixed this.